src/common/styles/icons.m.css

```diff
@@ -0,0 +1,148 @@
+@font-face {
+	font-family: 'dojo2';
+	src:
+		url('fonts/dojo2.ttf?k76fx7') format('truetype'),
+		url('fonts/dojo2.woff?k76fx7') format('woff'),
```

Comment:
Can we get woff2 icon file? It's getting better support, and there are some online converters if whatever we're using to generate the font files doesn't output it.